docs/microbit.rst

```diff
@@ -31,6 +31,22 @@ Functions
     :returns: The number of milliseconds since the board was switched on or
         restarted.
 
+.. py:function:: scale(value, from, to)
```

Comment:
Unfortunately `from` is a keyword in Python so cannot be used as an argument name.

How about `from_`?

Reply:
Yeah, I can't think of a better word, so `from_` looks like it might be the best option. 👍

Reply:
OK, I'll use `from_`.